frontend/lib/src/util/utils.ts

```diff
@@ -57,6 +57,9 @@ export const EMBED_SHOW_FOOTER = "show_footer"
 export const EMBED_LIGHT_THEME = "light_theme"
 export const EMBED_DARK_THEME = "dark_theme"
 export const EMBED_TRUE = "true"
+export const EMBED_HIDE_LOADING_SCREEN = "hide_loading_screen"
+export const EMBED_SHOW_LOADING_SCREEN_V1 = "show_loading_screen_v1"
+export const EMBED_SHOW_LOADING_SCREEN_V2 = "show_loading_screen_v2"
```

Comment:
Added these embed options, so we can turn on the old behavior, or even remove the loading skeleton altogether.

This part of the code needs a fresh review.